src/main/resources/static/test-scheme.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE Article (
+  id INTEGER NOT NULL IDENTITY,
+  title VARCHAR(45) NOT NULL,
+  label VARCHAR(45) NOT NULL,
+  text VARCHAR(200) NOT NULL,
+  created_date DATE NOT NULL,
+  author_id INT NOT NULL,
+  PRIMARY KEY (id));
```

Comment:
we use modified_date and created_date in a lot of situations so please add them to index